asv/benchmarks.py

```diff
@@ -198,7 +198,7 @@ def iter_unique(iter):
                     try:
                         with open(result_file, 'r') as fp:
                             benchmarks = json.load(fp)
-                    except (IOError, ValueError):
+                    except (OSError, ValueError):
```

Comment:
I think these should be split into two at some point. The errors are very different, one is that the expected file cannot be found, the other is something else (what?) that fails during discovery.